lib/ecto/migration.ex

```diff
@@ -274,6 +274,27 @@ defmodule Ecto.Migration do
 
           config :app, App.Repo, migration_default_prefix: "my_prefix"
 
+  ## Collations
+
+  For columns with a text type, the collation can be set on the column with the
+  option `:collation`. This can be useful when relying on ASCII sorting of
+  characters when using a fractional index for example. All supported collations
+  are not known by `ecto_sql` and specifying an incorrect collation might cause
+  a migration to fail.
+
+  ### N.B. be sure to match the collation on any columns that reference other text columns. See example below
```

Comment:
Let’s convert this to a regular paragraph or admonition. Please also avoid abbreviations such as NB, i.e. and similar :)

Reply:
Will do!